src/Tests/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -21,14 +21,18 @@ public TestProject([CallerMemberName] string name = null)
         }
 
         /// <summary>
-        /// A custom nametag for the project that's appended to a root test folder. By default, it is the unhashed name of the function that instantiated the TestProject object.
+        /// A name for the test project that's used to isolate it from a test's root folder by appending it to the root test path.
+        /// By default, it is the unhashed name of the function that instantiated the TestProject object.
         /// </summary>
         public string Name { get; set; }
 
         public bool IsSdkProject { get; set; } = true;
 
         public bool IsExe { get; set; }
 
+        /// <summary>
+        /// The project is a WPF or Windows Form App Executable.
```

Comment:
This actually just controls whether the `OutputType` in the generated project is set to `WinExe`.  It's not directly tied to the UI framework (WPF, Windows Forms, WinUI, or something else).